lib/functions/rootfs/distro-agnostic.sh

```diff
@@ -488,6 +488,9 @@ function install_distribution_agnostic() {
 	# permit root login via SSH for the first boot
 	sed -i 's/#\?PermitRootLogin .*/PermitRootLogin yes/' "${SDCARD}"/etc/ssh/sshd_config
 
+	# reduce clutter - disable showing last log when connected via SSH
+	sed -i 's/#\?PrintLastLog .*/PrintLastLog no/' "${SDCARD}"/etc/ssh/sshd_config
```

Comment:
This has nothing to do with the other changes.

Reply:
This is in this pr by mistake ...